cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/cache.go

```diff
@@ -342,36 +343,39 @@ func (c NamespacedResourceWatcherCache) onAddOrModify(add bool, unstructuredObj
 		return nil
 	}
 
+	var oldValue []byte
+	if !add {
+		oldValue, _ = c.redisCli.Get(c.redisCli.Context(), key).Bytes()
```

Comment:
Not sure if this is ignoring any error, or something else?

Reply:
good point. thanks. Will fix